src/vos/tests/vts_wal.c

```diff
@@ -526,7 +526,6 @@ wal_kv_large(void **state)
 static void
 wal_args_reset(struct io_test_args *args)
 {
-	sleep(1); /** See DAOS-13040 */
```

Comment:
I thought I've removed these two sleeps in my PR, but seems the landed one is bit different with what I pushed.

Reply:
Sorry, if that's the case.   It didn't rebase automatically when I landed my patch so I downloaded yours, did a merge with feature/vos_on_blob and pushed it.  So perhaps git removed them for some reason